pandas/core/reshape/tile.py

```diff
@@ -65,6 +65,12 @@ def cut(x, bins, right=True, labels=None, retbins=False, precision=3,
         The precision at which to store and display the bins labels.
     include_lowest : bool, default False
         Whether the first interval should be left-inclusive or not.
+    duplicates : {default 'raise', 'drop'}, optional
+        If bin edges are not unique, raise ValueError("Bin edges must be 
```

Comment:
you dont need to show the error message here.

Reply:
instead add a Returns section in the doc-string, showing that a ValueError can be raised